sunpy/sun/models.py

```diff
@@ -20,6 +20,7 @@
 
 import pandas
 from astropy.units import Quantity
+from astropy.table import Table
```

Comment:
I was wondering if using `QTable` would be better.

Reply:
I'm new to this so I don't know. If you think QTable would be better then I could try to use it and we can see which one works better.

Reply:
Yeah its just to try it. QTable is meant for tables with units as per my understanding.